src/js/hooks/cycleCount/useToResolveTab.jsx

```diff
@@ -323,9 +326,18 @@ const useToResolveTab = ({
     });
   };
 
-  const printResolveForm = () => {
-    // TODO: implement me!
-    console.log('print resolve form pressed');
+  const printResolveForm = async (fileFormat) => {
+    spinner.show();
+    const payload = {
+      requests: checkedCheckboxes.map((cycleCountRequestId) => ({
+        cycleCountRequest: cycleCountRequestId,
+        countIndex: 1, // We only ever allow for a single recount, so index is always 1.
+      })),
+    };
+    const response = await cycleCountApi.startRecount(payload, currentLocation?.id, fileFormat, { responseType: 'blob' });
```

Comment:
how does `responseType: 'blob'` work? Is that how we specify PDF?

Reply:
This is `axios`'s property, this needs to be set to be able properly download the file.